<launch>
    <node name="brain" pkg="agv_robot" type="brain_node.py" output="screen"/>
    <node name="camera" pkg="agv_robot" type="camera_node.py"/>
    <node name="trycicle_controller" pkg="agv_robot" type="trycicle_controller_node.py"/> 
    <node name="line_detector" pkg="agv_robot" type="line_detector_node.py"/>  
    <node name="rplidarNode" pkg="rplidar_ros"  type="rplidarNode">
    	<param name="serial_port" type="string" value="/dev/rplidar"/>
  	<param name="serial_baudrate"     type="int"    value="115200"/>
  	<param name="frame_id"            type="string" value="laser"/>
  	<param name="inverted"            type="bool"   value="false"/>
  	<param name="angle_compensate"    type="bool"   value="true"/>
    </node>
    <node name="object_detector" pkg="agv_robot" type="object_detector_node.py"/>
    <node name="rosserial"          pkg="rosserial_python"  type="serial_node.py">
    	<param name="port" type="string" value="/dev/ttyACM0"/>
        <param name="baud" type="string" value="57600"/>
    </node>
</launch>
